# Statistical Population Genomics

This is the website for Statistical Population Genomics seminar at the University of Connecticut (Spring 2025) led by Brandon Lind and Kent Holsinger. We are focusing on chapters from the book [*Statistical Population Genomics*](https://link.springer.com/book/10.1007/978-1-0716-0199-0) (free PDF; Springer 2020). Chapter numbers in the schedule below refer to chapters in the book. We will provide links to any papers we discuss that aren't in the book, and you'll find supplementary readings on the [Resources](resources.html) page. 

Please put your name and email address in this [Google sheet](https://docs.google.com/spreadsheets/d/1IUaW7QrRRxHV2oVKUZkdX5fSM4Q6vL_kGrVVRLH1Clo/edit?usp=sharing) to ensure that you receive any emails we send about the seminar.

**Note**: We expect students who have signed up for credit to lead the discussion one week.

## [Resources](resources.html)

## Schedule

|Date|Topic|Presenter|
|----|-----|---------|
|23 January|Introduction and organization| Brandon and Kent|
|30 January|Lexicon (Barosso et al., Chapter 1) and Adaptation in maize (Lorant et al., Chapter 12)| Brandon and Kent|
|6 February|Population admixture including Admixture and PCA (Liu et al., Chapter 4)<br />[Rosenberg et al. (2002)](https://dx.doi.org/10.1126/science.1078311)<br />[Novembre et al. (2008)](https://doi.org/10.1038/nature07331) |Kent|
|13 February|Population admixture including $`f_3`$ and $`f_4`$ statistics <br >[Raghavan et al. (2014) Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans](https://www.nature.com/articles/nature12736)|Brandon|
|20 February|[Melton et al. (2025) Genomics-Driven Monitoring of Fraxinus latifolia (Oregon Ash) to Inform Conservation and EAB-Resistance Breeding](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.17640)|Vidya|
|27 February|Detecting selection (Chapter 5)<br />[Nuismer et al. (2017) Identifying coevolving loci using interspecific genetic correlations](https://dx.doi.org/10.1002/ece3.3107)|Emma|
|6 March|Inferring the distribution of fitness effects (polyDFE; Chapter 6, sections 1 & 2.1)<br />[Huber et al. (2017)](https://www.pnas.org/doi/pdf/10.1073/pnas.1619508114)|Fahad|
|13 March|[Takahagi et al. (2016) Analysis of single nucleotide polymorphisms based on RNA sequencing data of diverse bio-geographical accessions in barley](https://www.nature.com/articles/srep33199)|Amee|
|20 March|***Spring break***|
|27 March|Coalescent simulation (Chapter 9)|(tentative)|
|3 April|Coalescent hidden Markov models (Chapter 8)<br />[Li and Durbin (2011) Inference of human population history from individual whole-genome sequences](https://dx.doi.org/10.1038/nature10231)<br />[Smith et al. (2018) Demographic inference in barn swallows using whole-genome data shows signal for bottleneck and subspecies differentiation during the Holocene](https://dx.doi.org/10.1111/mec.14854)|Cindy|
|10 April|Multiple sequentially Markovian coalescent (Chapter 7)<br />[Mather et al. (2020)](https://doi.org/10.1002/ece3.5888)|Analisa|
|17 April|SMC++ [(Terhorst et al. 2017) Robust and scalable inference of population history from hundreds of unphased whole genomes](https://doi.org/10.1038/ng.3748)|Cynthia|
|24 April|Tree sequences - [Kelleher et al. (2018) Efficient pedigree recording for fast population genetics simulation](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006581) <br />â€¢Ancestral recombination graphs (Chapter 10)|Brandon|
|1 May|Time-stratified ancestry analysis [Speidel et al. (2025) High-resolution genomic history of early medieval Europe](https://doi.org/10.1038/s41586-024-08275-2)|Kent|

