# Resources

## Slides

|Date|
|----|
|[30 January](lecture_slides/2025-01-30_chapter-12.pdf)|
|[6 February](lecture_slides/2025-02-06.pdf)|

## Software
* [Admixture](https://dalexander.github.io/admixture/) - available for Linux and OS X
* [angsd](https://www.popgen.dk/angsd/index.php/ANGSD) - analysis of sequence data taking genotype uncertainty into account
* [EEMS](https://github.com/dipetkov/eems) - ***E***stimated ***E***ffective ***M***igration ***S***urfaces
* [EIGENSOFT](https://github.com/DReichLab/EIG) - EIGENSOFT package for genomic data (including smartpca, Linux only)
* [PCAviz](https://github.com/NovembreLab/PCAviz) - Visualization tools for the results of a PCA on genomic data
* [smartsnp](https://christianhuber.github.io/smartsnp/index.html) - PCA analysis of genomic data in R
* [Structure](https://web.stanford.edu/group/pritchardlab/structure.html) - available for Linux, OS X, and Windoze
* [tskit](https://tskit.dev/) - software for tree sequences

## Reading

|Date|Article|
|----|-------|
|6 February|Bick, A. G. et al. 2024. Genomic data in the All of Us Research Program. Nature 627:340-346. [https://doi.org/10.1038/s41586-023-06957-x](https://doi.org/10.1038/s41586-023-06957-x)|
| |Kozlov, M. 2024. ‘All of Us’ genetics chart stirs unease over controversial depiction of race. Nature. [https://doi.org/10.1038/d41586-024-00568-w](https://doi.org/10.1038/d41586-024-00568-w)|
| |Lawson, D. J., L. van Dorp and D. Falush. 2018. A tutorial on how not to over-interpret STRUCTURE and ADMIXTURE bar plots. Nature Communications 9:3258. [https://doi.org/10.1038/s41467-018-05257-7](https://doi.org/10.1038/s41467-018-05257-7)|
| |Novembre, J., T. Johnson, K. Bryc, Z. Kutalik, A. R. Boyko, A. Auton, A. Indap, K. S. King, S. Bergmann, M. R. Nelson, M. Stephens and C. D. Bustamante. 2008. Genes mirror geography within Europe. Nature 456:98-101. [https://doi.org/10.1038/nature07331](https://doi.org/10.1038/nature07331)|
| |Rosenberg, N. A., J. K. Pritchard, J. L. Weber, H. M. Cann, K. K. Kidd, L. A. Zhivotovsky and M. W. Feldman. 2002. Genetic structure of human populations. Science 298:2381-2385. [https://dx.doi.org/10.1126/science.1078311](https://dx.doi.org/10.1126/science.1078311)|
| |Bradburd, G. S., G. M. Coop, and P. L. Ralph. 2018. Inferring Continuous and Discrete Population Genetic Structure Across Space. Genetics 210:genetics.301333.2018. [https://doi.org/10.1534/genetics.118.301333](https://doi.org/10.1534/genetics.118.301333)|
| | |
|13 February| Patterson, N., P. Moorjani, Y. Luo, S. Mallick, N. Rohland, Y. Zhan, T. Genschoreck, T. Webster and D. Reich. 2012. Ancient Admixture in Human History. Genetics 192:1065-1093. [https://dx.doi.org/10.1534/genetics.112.145037](https://dx.doi.org/10.1534/genetics.112.145037)|
| | Raghavan, M. et al. 2014. Upper Palaeolithic Siberian genome reveals dual ancestry of Native Americans. Nature 505(7481): 87-91. [https://dx.doi.org/10.1038/nature12736](https://dx.doi.org/10.1038/nature12736)|
| | Moreno-Mayar, et al. 2018. Terminal Pleistocene Alaskan genome reveals first founding population of Native Americans. Nature 553:203–207 [https://doi.org/10.1038/nature25173](https://doi.org/10.1038/nature25173)|
| | Peter, B. M. 2016. Admixture, Population Structure, and F-Statistics. Genetics 202:1485–1501. [https://doi.org/10.1534/genetics.115.183913](https://doi.org/10.1534/genetics.115.183913)|
| | Lipson M. 2020. Interpreting f-statistics and admixture graphs: theory and examples. [https://www.preprints.org/manuscript/202003.0237/v1](https://www.preprints.org/manuscript/202003.0237/v1)|
| | |
|20 February|Petsikova, D., J. Novembre, and M. Stephens.  2015.  Visualizing spatial population structure with estimated effective migration surfaces.  Nature Genetics. 48:94-100 [https://doi.org/10.1038/ng.3464](https://doi.org/10.1038/ng.3464)|
| |Shastry, V., M. Musiani, and J. Novembre. 2025. Jointly representing long-range genetic similarity and spatially heterogeneous isolation-by-distance. bioRxiv [https://doi.org/10.1101/2025.02.10.637386](https://doi.org/10.1101/2025.02.10.637386)
| | |
|27 February|Nuismer, S. L., C. E. Jenkins, and M. F. Dybdahl.  2017.  Identifying coevolving loci using interspecific genetic correlations. Ecology and Evolution.  7:6894-6903. [https://dx.doi.org/10.1002/ece3.3107](https://dx.doi.org/10.1002/ece3.3107)|
| | |
|6 March|Huber, C. D., B. Y. Kim, C. D. Marsden and K. E. Lohmueller. 2017. Determining the factors driving selective effects of new nonsynonymous mutations. Proceedings of the National Academy of Sciences 114:4465-4470. [https://doi.org/10.1073/pnas.1619508114](https://doi.org/10.1073/pnas.1619508114)|
| |Tataru, P., M. Mollion, S. Glémin and T. Bataillon. 2017. Inference of Distribution of Fitness Effects and Proportion of Adaptive Substitutions from Polymorphism Data. Genetics 207:1103-1119.  [https://doi.org/10.1534/genetics.117.300323](https://doi.org/10.1534/genetics.117.300323)
| | |
|13 March| |
| | |
|20 March|***Spring break***|
| | |
|27 March| |
| | |
|3 April|Li, H., and R. Durbin. 2011. Inference of human population history from individual whole-genome sequences.Nature 475:493-496. [https://dx.doi.org/10.1038/nature10231](https://dx.doi.org/10.1038/nature10231) |
| |Smith, C. C. R., S. M. Flaxman, E. S. C. Scordato, N. C. Kane, A. K. Hund, B. M. Sheta and R. J. Safran. 2018. Demographic inference in barn swallows using whole‐genome data shows signal for bottleneck and subspecies differentiation during the Holocene. Molecular Ecology 27:4200-4212. [https://dx.doi.org/10.1111/mec.14854](https://dx.doi.org/10.1111/mec.14854)|
| | |
|10 April|Mather, N., S. M. Traves and S. Y. W. Ho. 2020. A practical introduction to sequentially Markovian coalescent methods for estimating demographic history from genomic data. Ecology and Evolution 10:579-589. [https://doi.org/10.1002/ece3.5888](https://doi.org/10.1002/ece3.5888)|
| | |
|17 April|Terhorst, J., J. A. Kamm and Y. S. Song. 2017. Robust and scalable inference of population history from hundreds of unphased whole genomes. Nature Genetics 49:303-309. [https://doi.org/10.1038/ng.3748](https://doi.org/10.1038/ng.3748) |
|17 April| |
| | |
|24 April| |
| | |
|1 May|Speidel, L., M. Silva, T. Booth, B. Raffield, K. Anastasiadou, C. Barrington, A. Götherström, P. Heather and P. Skoglund. 2025. High-resolution genomic history of early medieval Europe. Nature 637:118-126. [https://doi.org/10.1038/s41586-024-08275-2](https://doi.org/10.1038/s41586-024-08275-2)


