\documentclass[titlepage,landscape]{seminar}
\usepackage{url}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{epstopdf}
\usepackage{slides}

\newcommand{\frack}{\frac{1}{k}}

\begin{document}

\myslide{
\heading{Individual assignment using {\tt STRUCTURE}}
  
\begin{eqnarray*}
\mbox{P}(i|k) &=& \frac{\mbox{P}(x_i|\gamma_k)}{\sum_k
  \mbox{P}(x_i|\gamma_k)} \\
x_i &=& \mbox{genotype of individual $i$} \\
\gamma_k &=& \mbox{genotype frequencies in population $k$}
\end{eqnarray*}
For example, if $A_1A_1$ is labeled as $1$, $A_1A_2$ as $2$, $A_2A_2$
as 3, and we assume that genotypes are in Hardy-Weinberg, then
\begin{eqnarray*}
\mbox{P}((1,2,2,1,3)|(p_{k1}, p_{k2}, p_{k3}, p_{k4}, p_{k5})) = (p_{k1}^2)(2p_{k2}q_{k2})(2p_{k3}q_{k3})(p_{k4}^2)(q_{k5}^2)
\end{eqnarray*}
}

\myslide{
\heading{Using {\tt STRUCTURE} in barberry}
  
{\it Berberis thunbergii\/}
\begin{itemize}
\item 85 feral, 7 horticultural, 4 cultivated
\item 147 polymorphic AFLP markers
\end{itemize}
\begin{table}
\begin{center}
\begin{tabular}{cc}
\hline\hline
K & Mean L(K) \\
\hline
2 & -2553.2 \\
3 & {\bf -2331.9} \\
4 & -2402.9 \\
5 & -2476.3 \\
\hline
\end{tabular}
\end{center}
\caption{Mean log probability of the data for $K=2,3,4,5$ in the {\it
    Berberis thunbergii\/} data}
\end{table}
}

\myslide{
\heading{Using {\tt STRUCTURE} in barberry}

  \begin{figure}
\resizebox{\textwidth}{!}{\includegraphics{lubell-structure.eps}}
\caption{Analysis of AFLP data from {\it Berberis
    thunbergii}}
\end{figure}
}

\myslide{
\heading{Using {\tt STRUCTURE} in humans}
  
\begin{itemize}

\item Human Genome Diversity Cell Line Panel (HGDP-CEPH)

\item 1056 individuals, 52 geographic populations, 377 autosomal
  microsatellite loci

\end{itemize}

\begin{figure}
\resizebox{\textwidth}{!}{\includegraphics{HGDP-CEPH.eps}}
\end{figure}

}

\myslide{
\heading{Principal components analysis of genotypes}

Principal components analysis is a ``dimension reduction'' method, a
way of reducing a very large number of variables to a smaller, more
manageable number for interpretation and analysis.

\begin{center}
  \includegraphics[height=5cm]{pca-example.eps}
\end{center}

\vfill\eject
}

\myslide{
\heading{Principal components analysis of genotypes}

3129 Europeans, 500,568 SNP loci

\begin{figure}
\begin{center}
\resizebox{0.5\textwidth}{!}{\includegraphics{human-PCA.eps}}
\end{center}
\end{figure}

}

\myslide{
  \heading{Combining approaches: {\it Protea repens}}

  \vfill

\begin{figure}
  \resizebox{!}{0.5\textheight}{\includegraphics{_KEH6704.png}}
  \resizebox{!}{0.5\textheight}{\includegraphics{repens-distribution.png}}
\end{figure}

}

\myslide{
  \heading{Structure}

  \vfill

  \begin{figure}
    \resizebox{\textwidth}{!}{\includegraphics{repens-structure.png}}
  \end{figure}
  
}

\myslide{
  \heading{Principal Components Analysis}

  \vfill

  \begin{figure}
    \begin{center}
    \resizebox{!}{0.8\textheight}{\includegraphics{pca-populations.png}}
    \end{center}
  \end{figure}
}
  
\myslide{
  \heading{Principal Components Analysis}

  \vfill

  \begin{figure}
    \begin{center}
    \resizebox{!}{0.8\textheight}{\includegraphics{pca-3-groups.png}}
    \end{center}
  \end{figure}
}
  
\end{document}



